app/stac_api/serializers.py

```diff
@@ -238,19 +260,24 @@ class Meta:
         ]
         # crs not in sample data, but in specs..
 
-    crs = serializers.SerializerMethodField()
+    # NOTE: when explicitely declaring fields, we need to had the validation as for the field
+    # in model !
+    id = serializers.CharField(
+        required=True,
+        max_length=255,
+        source="name",
+        validators=[validate_name, UniqueValidator(queryset=Collection.objects.all())]
+    )
+    links = CollectionLinkSerializer(required=False, many=True)
```

Comment:
according to specs, links are required (at least `href` and `rel` are required)

Reply:
Yes but, links are automatically generated, here we provide the ability to add extra non-generated links but these ones are not required. I'll add a comment though.